includes/Upgrade/Upgrades/V_3_3_1.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace WeDevs\Dokan\Upgrade\Upgrades;
+
+use WeDevs\Dokan\Abstracts\DokanUpgrader;
+
+class V_3_3_1 extends DokanUpgrader {
+
+    /**
+     * Updates withdraw database table column
+     *
+     * @since DOKAN_LITE_SINCE
+     *
+     * @return void
+     */
+    public static function update_withdraw_table_column() {
+        global $wpdb;
+
+        include_once ABSPATH . 'wp-admin/includes/upgrade.php';
+
+        $existing_columns = $wpdb->get_col( "DESC `{$wpdb->prefix}dokan_withdraw`", 0 );
+
+        if ( in_array( 'details', $existing_columns, true ) ) {
+            return;
+        }
+
+        $wpdb->query(
+            "ALTER TABLE `{$wpdb->prefix}dokan_withdraw` ADD COLUMN `details` longtext AFTER `note`" // phpcs:ignore
+        );
+    }
+
+    /**
+     * Updates withdraw database table column
+     *
+     * @since DOKAN_LITE_SINCE
```

Comment:
Same as above, instead of `DOKAN_LITE_SINCE`, please consider using possible upcoming version number.